pkg/providers/aws/provider_postgres.go

```diff
@@ -281,7 +281,8 @@ func (p *PostgresProvider) reconcileRDSInstance(ctx context.Context, cr *v1alpha
 	postgresPass := string(credSec.Data[defaultPostgresPasswordKey])
 	if postgresPass == "" {
 		msg := "unable to retrieve rds password"
-		return nil, croType.StatusMessage(msg), errorUtil.Wrap(err, msg)
+		return nil, croType.StatusMessage(msg), errorUtil.Errorf(msg)
+		//return nil, croType.StatusMessage(msg), errorUtil.Wrap(err, msg)
```

Comment:
Is there a reason to keep the commented out line ? :thinking:

Reply:
No reason to keep it. I must have forgotten to delete it 😄